tests/conftest.py

```diff
@@ -951,8 +949,8 @@ def cost_logger_fixture() -> Iterator[CostLogger]:
 
 @pytest_asyncio.fixture(scope="function")
 async def simulation(consensus_mode, bt):
-    if consensus_mode != Mode.PLAIN:
-        pytest.skip("Skipping this run. This test only supports one running at a time.")
+    #    if consensus_mode != Mode.PLAIN:
```

Comment:
i think we can delete this

Reply:
I'm going to go ahead and make this a raised concern since we at least shouldn't leave it sitting commented out.

Reply:
Removed